bigquery/README.md

```diff
@@ -135,6 +136,7 @@ export VM_ZONE=<the zone of the VM>
 export PROJECT=<project name>
 gcloud compute ssh $VM_NAME --project $PROJECT --zone $VM_ZONE --ssh-flag="-L" --ssh-flag="8080:localhost:8080" --ssh-flag="-L" --ssh-flag="8787:localhost:8787"
 ```
+If you are unable to connect, you may need to add a <a href="https://cloud.google.com/compute/docs/vpc/firewalls">firewall rule</a> for ports 22 and 80. 
```

Comment:
Did the port forwarding through ssh not work for some reason?  Atlas doesn't have good built in security, so it's not a good idea to open these ports in the firewall.

Reply:
Good point on the HTTP port. I may have overshot that. Maybe we just want 22, which would still needed to be open for SSH forwarding.

Reply:
It shouldn't need to be open at all if you use "gcloud compute ssh".  Did you have problems connecting using that?